### ISSUE TITLE
Run Build Task… vs. Ctrl+Shift+B

### ISSUE THREAD

**Issue by lvanvugt** [Microsoft/AL#3516]
In VSCode there seems a difference between Run Build Task… and Ctrl+Shift+B. Selecting Task > Run Build Task…  the command pallet urges me to enter to _Select build task to run_. But when I use the shortcut Ctrl+Shift+B it just builds. So what is the difference?

Reading https://code.visualstudio.com/docs/editor/tasks I get the idea the AL extension has implemented a default build task activated by the shortcut , but not selectable when using the menu item _Run Build Task_.

Anybody a clue?

**Comment by dennisfrostlander** [comment]
Yes, your understanding is correct and this is unfortunately a known limitation of the AL extension.
We override the shortcut Ctrl+Shift+B and run our compiler when you press it. A similar problem exists for F5 and starting the debugger from VSCode UI, see also https://github.com/Microsoft/AL/issues/262.

Recently, VSCode team has made certain improvements (partially) that will allow us make the experience more consistent, so we will soon revisit these issues.

**Comment by lvanvugt** [comment]
Clear and thanx.

**Comment by blrobl** [comment]
Hi, thank you for your suggestion! We are not prioritizing this for the immediate future, but we recommend you submit this at aka.ms/BCIdeas, or vote up the idea if its already there. We're constantly monitoring top Ideas and will consider them for a future release.

**Comment by SBalslev** [comment]
Thank you for raising this — we appreciate you taking the time to report it and we understand it matters to your workflow. After internal discussion, we've decided not to address this particular issue right now so we can focus on higher-impact items.

We value your feedback and encourage you to keep sharing your experiences with us. If this becomes more urgent or you have additional context, please don't hesitate to open a new issue.
